docs/content/v2.6/yugabyte-platform/security/enable-encryption-in-transit.md

```diff
@@ -4,7 +4,7 @@ headerTitle: Enable Encryption in Transit (TLS)
 linkTitle: Enable encryption in transit (TLS)
 description: Use Yugabyte Platform to enable encryption in transit (TLS) on a YugabyteDB universe and connect to clients.
 menu:
-  v2.6:
+  latest:
```

Comment:
Is this correct? Page not showing in left nav

Reply:
It's only showing in latest. Don't know why. Maybe it'll show when published. Will ask Alex